ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -98,6 +99,25 @@ def validate_expected_files(self):
                     if "kind" not in f.read().lower():
                         self.files_content_issue.append(file)
 
+    def verify_ceph_file_content(self):
+        """
+        Verify ceph command does not return an error
+        https://bugzilla.redhat.com/show_bug.cgi?id=2014849
+
+        """
+        if self.type_log != "CEPH":
+            return
+        for root, dirs, files in os.walk(self.root):
+            for file in files:
+                try:
+                    with open(f"{root}/{file}", "r") as f:
+                        data_file = f.read()
+                    exit_code_error = re.findall("exit code [1-9]+", data_file.lower())
```

Comment:
Since the _regular expression_ string is always the same, I will be better (more efficient) to [_compile_](https://docs.python.org/3/library/re.html#re.compile) it once before the loop and then use the compiled [_pattern_](https://docs.python.org/3/library/re.html#re.Pattern.findall). 
Something like this - before the loop:
```python
pattern = re.compile("exit code [1-9]+")
```
and then:
```python
              exit_code_error = pattern.findall(data_file.lower())
```

Reply:
done